example/src/DrawerExample.re

```diff
@@ -0,0 +1,29 @@
+open BsReactNavigation;
+open Config;
+
+module Drawer =
+  DrawerNavigation.Create({
+    type item = Config.item;
+
+    let items = [Dashbord, Settings];
+    let drawerOptions =
+      DrawerNavigation.drawerOptions(~activeTintColor="#847", ());
+
+    let order = [Dashbord, Settings];
+
+    let getItem = tab =>
+      switch (tab) {
+      | Dashbord => (
+          "Dashbord",
```

Comment:
Same for the "key" - it's a bit confusing to have both variants and strings. I would suggest we do what we are going to do in a TabNavigator and eventually share the same function across both - hashing.